src/models/Message.model.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../db.js';
+
+export const Message = sequelize.define(
+  'message',
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    roomId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    text: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    time: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    author: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    tableName: 'messages',
+    updatedAt: false,
+    createdAt: false,
+  },
+);
+
+Message.sync({ force: true });
```

Comment:
The `sync` method with the `force` option set to `true` will drop the table if it already exists and create a new one. This is not a good practice for production code as it would delete all your existing data. It's usually used for development or testing purposes. If you want to make sure the table is created, you can use `sync` without the `force` option. If you want to alter an existing table, you can use `sync` with the `alter` option set to `true`.